CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+## [2.20.1] - 2023-05-16
+
+## [2.20.1] - 2023-05-16
```

Comment:
duplicated

Reply:
Yeah, fixed this by manually merging. The old release automation sucks when you want to edit the release.